TFlippy_TerritoryControl_Core_Dev/Entities/FleshHit.as

```diff
@@ -121,89 +87,113 @@ f32 onHit(CBlob@ this, Vec2f worldPoint, Vec2f velocity, f32 damage, CBlob@ hitt
 	
 	if (this.hasTag("equipment support"))
 	{
-		if ((this.get_string("equipment_torso") == "bulletproofvest") && (customData == HittersTC::bullet_low_cal || customData == HittersTC::bullet_high_cal || customData == HittersTC::shotgun || customData == HittersTC::railgun_lance))
+		bool isBullet = (customData == HittersTC::bullet_low_cal || customData == HittersTC::bullet_high_cal || customData == HittersTC::shotgun || customData == HittersTC::railgun_lance);
+		// torso
+		if (this.get_string("equipment_torso") == "bulletproofvest" && customData != HittersTC::radiation)
 		{
-			f32 vestMaxHealth = 25.0f;
-			f32 vestHealth = vestMaxHealth - this.get_f32("bpv_health");
-			f32 ratio = vestHealth / vestMaxHealth;
+			f32 armorMaxHealth = 25.0f;
+			f32 armorHealth = armorMaxHealth - this.get_f32("bpv_health");
+			f32 ratio = armorHealth / armorMaxHealth;
 
 			switch (customData)
 			{
-				case HittersTC::bullet_low_cal:
-					ratio = ratio * 0.80f;
+				case HittersTC::bullet_low_cal: ratio *= 0.80f;
 					break;
-			
+
 				case HittersTC::bullet_high_cal:
-					ratio = ratio * 0.60f;
+				case HittersTC::railgun_lance:
+					ratio *= 0.60f;
 					break;
-					
-				case HittersTC::shotgun:
-					ratio = ratio * 0.85f;
+
+				case HittersTC::shotgun: ratio *= 0.85f;
```

Comment:
case shotgun:
 ration
 break;
 
imo keep this consistent because you already have 1 like it